cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -100,9 +100,9 @@ def test_single_vm_migration_power_state_tags_retirement(appliance, provider,
 @pytest.mark.parametrize('source_type, dest_type, template_type',
                          [
                              ['nfs', 'nfs', [Templates.RHEL7_MINIMAL,
-                                             Templates.UBUNTU16_TEMPLATE,
-                                             Templates.RHEL69_TEMPLATE,
-                                             Templates.WIN7_TEMPLATE]
+                                             Templates.RHEL7_MINIMAL,
+                                             Templates.RHEL7_MINIMAL,
+                                             Templates.RHEL7_MINIMAL]
```

Comment:
Why parametrize on the same template type multiple times?

Reply:
The `test_multi_host_multi_vm_migration`  test requires that multiple VMs be deployed. Here, 'template_type' is an argument being passed to the 'mapping_data_multiple_vm_obj_single_datastore' fixture which deploys multiple VMs for each template in 'template_type'.
Prior to this change, the test_multi_host_multi_vm_migration test was passing a list of four different templates to the 'mapping_data_multiple_vm_obj_single_datastore' fixture. The `RHEL7_MINIMAL' template is just 8GB, whereas the other templates vary in size from 16 - 32GB. A 32 GB VM takes much longer to be migrated to RHV .For the purpose of testing, it's sufficient to have multiple VMs deployed through any supported template(OS).